src/Univapay/Resources/PaymentData/ThreeDS.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Univapay\Resources\PaymentData;
+
+use Univapay\Enums\ThreeDSStatus;
+use Univapay\Resources\Jsonable;
+use Univapay\Utility\FormatterUtils;
+use Univapay\Utility\FunctionalUtils;
+use Univapay\Utility\Json\JsonSchema;
+
+class ThreeDS
+{
+    use Jsonable;
+
+    public $enabled;
+    public $status;
+    public $redirectEndpoint;
+    public $redirectId;
```

Comment:
We might want to rename the other one for charge/subscription as `PaymentThreeDS` and rename this one as `TokenThreeDS` for clarity and to prevent confusion.